.gitignore

```diff
@@ -54,3 +54,4 @@ _UpgradeReport_Files/
 Backup*/
 UpgradeLog*.XML
 UpgradeLog*.htm
+/NUnit-3.5.0
```

Comment:
`**/packages/*` should cover this already if you are using the project default version.